db/migrate/20210730192201_add_scored_business_unit_in_control_objective_item.rb

```diff
@@ -0,0 +1,6 @@
+class AddScoredBusinessUnitInControlObjectiveItem < ActiveRecord::Migration[6.0]
+  def change
+    add_column :control_objective_items, :scored_business_unit_id, :integer, null: true
+    add_foreign_key :control_objective_items, :business_units, column: :scored_business_unit_id
```

Comment:
Acá de jodido, pero se puede hacer algo como:

```suggestion
    change_table :control_objective_items do |t|
      t.references :scored_business_unit, index: true, foreign_key: FOREIGN_KEY_OPTIONS.dup
    end
```

Reply:
impecable

Reply:
impecable ya lo resolvimos por mail